types/src/account_config.rs

```diff
@@ -44,8 +44,14 @@ pub const HASH_MODULE_NAME: &str = "Hash";
 pub fn core_code_address() -> AccountAddress {
     AccountAddress::default()
 }
+
 pub fn association_address() -> AccountAddress {
-    AccountAddress::default()
+    let mut addr_array = [0x0; ADDRESS_LENGTH];
```

Comment:
You could lazy_static of `AccountAddress::from_hex_string("0xA550C18")`

Reply:
I just looked into this, but ended up feeling worse about having to unwrap the `Result` that `from_hex_string` returns than I did about my ugly array code.